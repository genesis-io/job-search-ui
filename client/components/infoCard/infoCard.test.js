import React from 'react';
import InfoCard from './infoCard';

test('render home component', () => {
  const wrapper = shallow(<InfoCard />);
  expect(wrapper).toMatchSnapshot();
});