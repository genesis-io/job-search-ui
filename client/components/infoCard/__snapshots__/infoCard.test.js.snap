// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render home component 1`] = `
<div
  className="info-card"
  style={
    Object {
      "backgroundColor": Array [
        undefined,
      ],
    }
  }
>
  <i
    className="base fa undefined"
  />
  <h1 />
  <p />
  <Link
    replace={false}
    style={
      Object {
        "color": Array [
          undefined,
        ],
      }
    }
  />
</div>
`;
